server/http.go

```diff
@@ -193,6 +193,30 @@ func (p *Plugin) completeUserOAuthToZoom(w http.ResponseWriter, r *http.Request)
 	}
 }
 
+func (p *Plugin) getMattermostUserIdFromZoomParticipant(participant zoom.Participant) (string, error) {
```

Comment:
@larkox Does the code in this function look correct?

Reply:
For JWT we should be able to get the user practically the same as we do with Account level App.

Nevertheless, we have to take in mind for any case that the user may not be in Mattermost (not all Zoom users need to be in Mattermost). Therefore, we should handle that beautifully somehow.